src/include/dns-server/dialog-main.rb

```diff
@@ -1876,38 +1870,72 @@ def HandleExpertZonesPage(key, event)
     end
 
     # Write settings dialog
-    # @return `abort if aborted and `next otherwise
+    #
+    # @return [Symbol] :next whether configuration is saved successfully
+    #                  :back if user decided to change settings
+    #                  :abort otherwise
     def WriteDialog
-      Wizard.RestoreHelp(Ops.get_string(@HELPS, "write", ""))
-      ret = DnsServer.Write
-      if ret
-        service.reload if service.running? && status_widget.reload_flag?
-        :next
-      else
-        if Popup.YesNo(_("Saving the configuration failed. Change the settings?"))
-          :back
-        else
-          :abort
-        end
-      end
+      log.info("Running write dialog")
```

Comment:
Do you really want to log this?

Reply:
No, I don't. However I started with `dhcp` module, which has an identical method which logs that information. So, in attempt to has an unified code I put it also here.

But no problem deleting it :)